src/ApiService.php

```diff
@@ -75,13 +75,32 @@ public function getPhpAgent(array $options = []) {
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getRumAgentScript(array $options = []) {
+    // Initialize and return RUM Agent script.
+    return 'elasticApm.init({
+      serviceName: ' . $this->config['rumAgent']['serviceName'] . ',
+      serverUrl: '. $this->config['rumAgent']['serverUrl'] . ',
+      serviceVersion: ' . $this->config['rumAgent']['serviceVersion'] . ',
+    })';
+  }
```

Comment:
**CHANGE REQUEST**

This is a bit confusing to read here within PHP with concatenations across lines. It can be written to be more readable for the developer and also without the line breaks that are unnecessary for the final script code.

```
return 'elasticApm.init({' .
  'serviceName:' . $this->config['rumAgent']['serviceName'],' .
  'serverUrl:' . $this->config['rumAgent']['serverUrl'],' .
  'serviceVersion:' . $this->config['rumAgent']['serviceVersion']' .
'});';
```

Notice that you're passing a JavaScript object, it's not a PHP array and the last item should not end with `,`.

Reply:
Done. @krystalcode I have modified the code as you suggested.